config/initializers/assets.rb

```diff
@@ -1,21 +1,21 @@
 # Be sure to restart your server when you modify this file.
 
-# Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = '1.0'
+Rails.application.config.assets.tap do |assets|
+  # Version of your assets, change this if you want to expire all your assets.
+  assets.version = '1.0'
 
-# Add additional assets to the asset load path
-# Rails.application.config.assets.paths << Emoji.images_path
+  # Add additional assets to the asset load path
+  # assets.paths << Emoji.images_path
 
-# Precompile additional assets.
-# application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-# Rails.application.config.assets.precompile += %w( search.js )
+  # Precompile additional assets.
+  # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
+  # assets.precompile += %w( search.js )
 
-Rails.application.config.assets.precompile += %w( embed.css )
+  assets.precompile += %w( embed.css )
 
-# MediaElement 4 files
-Rails.application.config.assets.precompile += %w( mejs4_player.js )
-Rails.application.config.assets.precompile += %w( mejs4_player.scss )
-
-# MediaElement 2 files
-Rails.application.config.assets.precompile += %w( mejs2_player.js )
-Rails.application.config.assets.precompile += %w( mejs2_player.scss )
+  # MediaElement 4 files
+  assets.precompile += %w( mejs4_player.js mejs4_player.scss )
+  assets.precompile += Dir[Rails.root + 'vendor/assets/stylesheets/mediaelement/mejs-controls.*']
```

Comment:
Does Rails.root always have a ending slash or would File.join be better?

Reply:
Rails.root is a Pathname, so addition acts like `File.join`. But I've found another issue with this PR so I'm going to close it and resubmit when I've got it worked out.